### ISSUE TITLE
[Cinematik]: Avoid Bumped torrents in RSS

### ISSUE THREAD

**Issue by leonard4546** [Jackett/Jackett#12929]
### Is there already a request for your feature?

- [X] I have checked older issues, open and closed

### Is your feature request related to a problem? Please describe.

if someone reseeds dead torrent on cinematik then that torrent gets bumped to main torrents page with tag "Bumped". is it possible to avoid these Bumped torrents in rss as I want to download only new torrents.

For Ex. https://files.catbox.moe/p67pwm.png this torrent uploaded 542 weeks ago was reseeded recently and then it was bumped to browse.php page

### Describe the solution you'd like

Option to Avoid Bumped torrents in indexer settings.

**Comment by garfield69** [comment]
Yes, we should be able to add an option on the indexer config to exclude torrents with that flag.
However this cannot be coded and tested currently as no one on the team has an account on this private site ATM.
If you send an invite to garfieldsixtynine -at- gmail.com I can add this feature to the indexer.

**Comment by leonard4546** [comment]
> Yes, we should be able to add an option on the indexer config to exclude torrents with that flag. However this cannot be coded and tested currently as no one on the team has an account on this private site ATM. If you send an invite to garfieldsixtynine -at- gmail.com I can add this feature to the indexer.

Sorry but it's basically impossible to get an invite

**Comment by garfield69** [comment]
In that case we can try to code using a dump of the HTML as a reference.
Please tick the enhanced logging checkbox on the Jackett dashboard and click the apply server settings button, 
then click the TEST button on the right of the cinematik indexer row or the configured indexers page.
Read https://github.com/Jackett/Jackett/wiki/Troubleshooting/#how-to-provide-a-jackett-enhanced-log-for-a-ticket on how to find the log and post the file here after redacting for privacy and hopefully that should provide the info we need to code a solve.
Albeit the chances of it not working have escalated since we cant test before publishing.

**Comment by garfield69** [comment]
Files received. Thank you.

**Comment by garfield69** [comment]
try out v0.20.535 and let me know how it goes.

**Comment by leonard4546** [comment]
> try out v0.20.535 and let me know how it goes.

Tried Search and RSS and it's working. Thank you so much that was fast. Can you keep this issue open for a day? I will update if there is any issue. 

Btw is there any possibility to add fetch RSS with username filter. Like it will fetch torrents from specific usernames mentioned in indexer settings. is it possible to add this as an optional feature?

**Comment by garfield69** [comment]
Can only add a feature if it supported by the web sites torrent search page. 
is there such a parm?
In jpg of the browse.php that you provided, I see Source, Genres, Special, Collections, Title and Director.

**Comment by leonard4546** [comment]
> Can only add a feature if it supported by the web sites torrent search page. is there such a parm? In jpg of the browse.php that you provided, I see Source, Genres, Special, Collections, Title and Director.

Yes. There is username above upload time. I blurred that because of privacy. 

https://files.catbox.moe/p67pwm.png in this image there is username above "542 Weeks Ago"

**Comment by garfield69** [comment]
yes, but that is a torrent detail not a search parameter. the parms are all the ones above the torrent list, the ones I mentioned earlier.

**Comment by leonard4546** [comment]
> yes, but that is a torrent detail not a search parameter. the parms are all the ones above the torrent list, the ones I mentioned earlier.

Ohh My bad. No we can't search with username.
